web_crawler.py

```diff
@@ -0,0 +1,106 @@
+"""search for urls and save their html content to files asynchronously"""
```

Comment:
[matan_webcrawler_traceback.pdf](https://github.com/matanhamelech/web-crawler/files/5859741/matan_webcrawler_traceback.pdf)


I get an error whenever I run the code. It saves some pages though.